### PR TITLE
Update ABOUT.md

### PR DESCRIPTION
**Pull request by Santiago00lp**
Update doc with benefits of LFE

### CODE REVIEW

File: docs/ABOUT.md [hunk 1]
```diff
@@ -1,8 +1,16 @@
 # About
 
-Lisp Flavoured Erlang (AKA LFE) is a lisp syntax front-end to the Erlang compiler. 
+**Lisp Flavoured Erlang** (AKA LFE) is a lisp syntax **front-end** to the **Erlang compiler**.
```
Comment (BNAndras) [inline docs/ABOUT.md:3]:
```suggestion
**Lisp Flavoured Erlang** (AKA LFE) is a Lisp syntax **front-end** to the **Erlang compiler**. 
```

File: docs/ABOUT.md [hunk 2]
```diff
@@ -1,8 +1,16 @@
 # About
 
-Lisp Flavoured Erlang (AKA LFE) is a lisp syntax front-end to the Erlang compiler. 
+**Lisp Flavoured Erlang** (AKA LFE) is a lisp syntax **front-end** to the **Erlang compiler**. 
 Code produced with it is compatible with "normal" Erlang code. 
 LFE is a (proper) Lisp based on the features and limitations of the Erlang VM.
+LFE is useful in applications that require the **robustness** and **concurrency** handling of Erlang, combined with the **flexibility** of a Lisp-family language.
+
+**Key Benefits:**
```
Comment (BNAndras) [inline docs/ABOUT.md:8]: We list out key features for LFE at https://github.com/exercism/lfe/blob/13ec9033ee17a056d7f9c2acb6e8fc7af813fa2b/config.json#L498. This appears on the track's about page at https://exercism.org/tracks/lfe/about underneath the contents of this file. There we shouldn't repeat that information here.

File: docs/ABOUT.md [hunk 3]
```diff
@@ -1,8 +1,16 @@
 # About
 
-Lisp Flavoured Erlang (AKA LFE) is a lisp syntax front-end to the Erlang compiler. 
+**Lisp Flavoured Erlang** (AKA LFE) is a lisp syntax **front-end** to the **Erlang compiler**. 
 Code produced with it is compatible with "normal" Erlang code.
```
Comment (BNAndras) [inline docs/ABOUT.md:4]: I think we should remove "normal" here since it's unclear what this means.

File: docs/ABOUT.md [hunk 4]
```diff
@@ -1,8 +1,16 @@
 # About
 
-Lisp Flavoured Erlang (AKA LFE) is a lisp syntax front-end to the Erlang compiler. 
+**Lisp Flavoured Erlang** (AKA LFE) is a lisp syntax **front-end** to the **Erlang compiler**. 
 Code produced with it is compatible with "normal" Erlang code. 
 LFE is a (proper) Lisp based on the features and limitations of the Erlang VM.
```
Comment (BNAndras) [inline docs/ABOUT.md:5]: This sentence is a bit vague. What is proper? What are the features and limitations of the Erlang VM? This doesn't add any information to the last two sentences so perhaps it's best to remove it.

File: docs/ABOUT.md [hunk 5]
```diff
@@ -1,8 +1,16 @@
...
+
+**Key Benefits:**
+- **Lisp Syntax**: LFE uses a syntax based on S-expressions (symbolic expressions), characteristic of Lisp-family languages. This allows easy manipulation of programs as data, a powerful feature for metaprogramming.
+- **Erlang Concurrency**: It leverages Erlang's actor-based concurrency model. Processes in Erlang are lightweight and can be used in large numbers without significant performance costs.
+- **Fault Tolerance**: LFE inherits Erlang's capability to create distributed and fault-tolerant systems. This includes features such as process restarts and supervisors to handle errors robustly.
+- **Interoperability**: It can easily integrate with existing Erlang code, allowing developers to use libraries and applications written in Erlang without issues.
+- **Lisp Macros**: Like other Lisp languages, LFE allows developers to write macros, which are programs that write other programs. This facilitates the creation of new abstractions and language extensions.
 
 LFE has many origins, depending upon whether you're looking at Lisp, Erlang, or LFE-proper.
```
Comment (BNAndras) [inline docs/ABOUT.md:15]: This sentence is a bit vague. Perhaps we should play up that LFE predates Elixir and the creator, Robert Virding, was a co-creator of Erlang itself. We could highlight that Common Lisp is a strong influence on LFE.

File: docs/ABOUT.md [hunk 6]
```diff
@@ -1,8 +1,16 @@
...
+
+**Key Benefits:**
+- **Lisp Syntax**: LFE uses a syntax based on S-expressions (symbolic expressions), characteristic of Lisp-family languages. This allows easy manipulation of programs as data, a powerful feature for metaprogramming.
+- **Erlang Concurrency**: It leverages Erlang's actor-based concurrency model. Processes in Erlang are lightweight and can be used in large numbers without significant performance costs.
+- **Fault Tolerance**: LFE inherits Erlang's capability to create distributed and fault-tolerant systems. This includes features such as process restarts and supervisors to handle errors robustly.
+- **Interoperability**: It can easily integrate with existing Erlang code, allowing developers to use libraries and applications written in Erlang without issues.
+- **Lisp Macros**: Like other Lisp languages, LFE allows developers to write macros, which are programs that write other programs. This facilitates the creation of new abstractions and language extensions.
 
 LFE has many origins, depending upon whether you're looking at Lisp, Erlang, or LFE-proper. 
 The LFE community of contributors embraces all of these and more.
```
Comment (BNAndras) [inline docs/ABOUT.md:16]: This sentence doesn't really describe how the origins are embraced so this sentence can be dropped entirely for the sake of brevity.

**Comment by Santiago00lp** [comment]
Thanks for the feedback, here is a new version.

File: docs/ABOUT.md [hunk 7]
```diff
@@ -1,12 +1,11 @@
 # About
 
-Lisp Flavoured Erlang (AKA LFE) is a lisp syntax front-end to the Erlang compiler. 
-Code produced with it is compatible with "normal" Erlang code. 
-LFE is a (proper) Lisp based on the features and limitations of the Erlang VM.
+**Lisp Flavoured Erlang** (AKA LFE) cmbine the robustness and concurrency of Erlang with the flexibility and power of a Lisp syntax.
```
Comment (kotp) [inline docs/ABOUT.md:3]:
```suggestion
**Lisp Flavoured Erlang** (AKA LFE) combine the robustness and concurrency of Erlang with the flexibility and power of a Lisp syntax.
```

File: docs/ABOUT.md [hunk 8]
```diff
@@ -1,12 +1,11 @@
...
 
-LFE is a Lisp-2. 
-In Lisp-2, the rules for evaluation in the functional position of a form are distinct from those for evaluation in the argument positions of the form. 
-In addition, LFE not only has separate value and function space but also allows multiple function definitions for the same name, as Erlang does.
+LFE is useful in applications that require the **robustness** and **concurrency** handling of Erlang, combined with the **flexibility** of a Lisp-family language.
+
+LFE is classified as a Lisp-2, which means it has separate evaluation rules for functions and arguments. This design allows for more sophisticated function definitions and greater flexibility in code structure. Moreover, LFE supports polymorphic function definitions, as Erlang does.
+
+LFE predates Elixir and was created by Robert Virding, one of the original developers of Erlang. With strong influences from Common Lisp, LFE is built on a foundation of decades of programming language development.
```
Comment (kotp) [inline docs/ABOUT.md:11]:
```suggestion
LFE predates Elixir and was created by Robert Virding, one of the original developers of Erlang. With strong influences from Common Lisp, LFE is built on a foundation of decades of programming language development.

```

**Review by kotp — CHANGES_REQUESTED** [review]
Line ending and typo-graphical error.

**Comment by BNAndras** [comment]
Looks like the CI is flaking again. I’ll take a look at binary this weekend.

**Comment by BNAndras** [comment]
Once merged, #254 should fix the CI issue.

**Review by kotp — APPROVED** [review]

**Pull request merged**